.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: scala
+
+scala:
+  - "2.12.3"
```

Comment:
Shouldn't this depend on the sbt version we target?

Reply:
`^^` overrides it. So this had no real effect. I didn't feel confortable not specifying any version here, though, as Travis does not document what will happen if you specify no Scala version.

Reply:
Hmmm... I see. This is very misleading when looked at. Could you:

- Add it to `crossScalaVersions` in `build.sbt`.
- Consider using `matrix` to set both the sbt and scala version, making this explicit. Like so:

```
matrix:
  include:
    - scala: 2.13.3
      env: SBT_VERSION=1.0.2
    - scala: 2.10.6
      env: SBT_VERSION=0.13.16
script:
  - sbt "^^$SBT_VERSION" ++$TRAVIS_SCALA_VERSION publishLocal scripted
  ```

This is only a couple of lines more, but makes it much more explicit what happens (and will display properly in the Travis UI).